Fortnox.NET/WebSockets/FortnoxWebSocketClient.cs

```diff
@@ -80,7 +80,7 @@ private async Task Send(string message, CancellationToken cancellationToken = de
             await _webSocket.SendAsync(messageBuffer, WebSocketMessageType.Text, true, cancellationToken);
```

Comment:
There is an error being thrown with an error message that is misspelled "recieve" in this file